tests/src/test/java/alluxio/IntegrationTestUtils.java

```diff
@@ -77,5 +85,55 @@ public static String randomString() {
     return sb.toString();
   }
 
+  /**
+   * Triggers two heartbeats to wait for a given list of blocks to be removed from both master and
+   * worker.
+   * Blocks until the master and block are in sync with the state of the blocks.
+   *
+   * @param bw the block worker that will remove the blocks
+   * @param blockIds a list of blockIds to be removed
+   */
+  public static void waitForBlocksToBeRemoved(final BlockWorker bw, long... blockIds) {
```

Comment:
maybe rename to `waitForBlocksToBeFreed` to make it clear that removed != deleted